Sources/PointFree/Emails/SendEmail.swift

```diff
@@ -13,7 +13,7 @@ func sendEmail(
   -> EitherIO<Error, Mailgun.SendEmailResponse> {
 
     let (plain, html): (String, String?) =
-      destructure(
+      either3(
```

Comment:
Not sure if we wanna further generalize these in the future. But it seemed appropriate.